keadm/cmd/keadm/app/cmd/util/common.go

```diff
@@ -46,7 +46,7 @@ import (
 )
 
 var (
-	kubeReleaseRegex = regexp.MustCompile(`^v?(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)([-0-9a-zA-Z_\.+]*)?$`)
```

Comment:
Why delete the `\` here?  We just import it from kubernetes, and it's also changed in K8s, ref to this patch: https://github.com/kubernetes/kubernetes/pull/112241    @cl2017

Reply:
ok, I had changed it from k8s